go.mod

```diff
@@ -21,6 +21,11 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```

Comment:
Why are you creating yet one `required` block?

Reply:
fixed